### PR TITLE
Add supported GUI environments

### PR DESCRIPTION
**Pull request by pamasse**
Based on manual tests, i create a first version of tested GUI configurations (OS + Browser).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,20 @@ Ubuntu 18.04     | 5.26
...
+### Supported Client Browser versions
+
+The Zonemaster GUI is tested against these configurations and should work perfectly with close configuration.
+
+Operating System | Browser | Version
+---------------- | ------- | -------
+Ubuntu 18.04     | Firefox | 64+
+Ubuntu 18.04     | Chrome  | 64+
+Widnows          | Firefox | 64+
+Widnows          | Chrome  | 64+
```
Comment (matsduf) [inline README.md]: The versions should be the specific versions that we have tested (are going to test). In my Mac I have version 65 of Firefox and 73 of Chrome.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,22 @@ Ubuntu 18.04     | 5.26
 Zonemaster requieres Perl version 5.14.2 or higher. Zonemaster has been 
 tested with the default version of Perl in the OSs as listed in the table above.
 
+### Supported Client Browser versions
+
+The Zonemaster GUI is tested against these configurations and should work perfectly with close configuration.
+
```
Comment (matsduf) [inline README.md]: "tested against these configurations" --> "tested against the browers below and for each browser the stated version and under the listed OS"

Is "close configuration" correct English? What does it mean?

File: README.md [hunk 3]
```diff
@@ -82,6 +82,22 @@ Ubuntu 18.04     | 5.26
...
+Operating System | Browser | Version
+---------------- | ------- | -------
+Ubuntu 18.04     | Firefox | 64, 65, 66
+Ubuntu 18.04     | Chrome  | 66
+Windows 10       | Firefox | 64, 65, 66
+Windows 10       | Chrome  | 73
+MacOs            | Firefox | 65
+MacOs            | Chrome  | 73
+
+Zonemaster was tested manually or with testing tools.
```
Comment (matsduf) [inline README.md]: "Zonemaster" --> "Zonemaster GUI"

**Comment by sandoche2k** [comment]
@matsduf not related to the PR from Pierre. But there is a line in the readme, which states the follows: 
```
*) MariaDB, not MySQL
```
which I think should be completed

**Comment by sandoche2k** [comment]
@pamasse merge?

**Pull request merged**